content/post/what-is-production.md

```diff
@@ -10,51 +10,58 @@ tags:
  - Architecture
 ---
 
-I have spoken to many fellow analytics practitioners who are either adament that they want their team to *never* touch "production." While there are good reasons to be careful whenever you make changes that could impact customers, I believe that as software becomes more data-driven it is critical to find safe ways to empower Analytics teams to build and deploy data-driven applications.
+I have spoken to many fellow analytics practitioners who are adament that they want their team to *never* touch "production." While there are good reasons to be careful whenever you make changes that could impact customers, I believe that as software becomes more data-driven it is critical to find safe ways to empower Analytics teams to build and deploy data-driven applications.
 
 <!--more-->
 
+
+## What is Production?
+
 First, some term definition. “Production” is an environment. Commonly there are a few environments where code may be run (in order of progression):
 
 * Local development
 * Remote development
 * Staging / QA
 * Production
 
-Every “application” will have multiple environments, one of which will be the one facing non-developer users. The one facing non-developer users is typically called “production”. Before we get too into the weeds, I want to start with my boldest claim: all of your analysts are deploying to production every day, already. Let me walk you through my favorite[^1] line of questioning:
+Every “application” will have multiple environments, one of which will be the one facing non-developer users. The one facing non-developer users is typically called “production”. 
+
+Here's my thesis: It's critical for Analytics teams to impact customer facing applications. If it is not possible to enable that in a safe way, then you should modify your customer facing application in such a way to make that possible.
```

Comment:
Do you want to add a qualifier such as "If you believe that Analytics should play a role in your customer facing app"? I can imagine organizations that have customer facing apps that aren't data-driven in any meaningful way.

But okay to ignore this if you think it muddies the water or you address it further on, which I think you might.

Reply:
Good call. I'm going to drop a footnote.